data/World/Shadow Temple MQ.json

```diff
@@ -153,7 +183,17 @@
         "region_name": "Shadow Temple Beyond Boat",
         "dungeon": "Shadow Temple",
         "locations": {
-            "Shadow Temple MQ GS After Ship": "Hookshot"
+            "Shadow Temple MQ GS After Ship": "Hookshot",
+            "Shadow Temple MQ After Boat Pot 1": "True",
+            "Shadow Temple MQ After Boat Pot 2": "True",
+            "Shadow Temple MQ After Boat Upper Recovery Heart 1" : "Bow and can_play(Song_of_Time) and Longshot",
+            "Shadow Temple MQ After Boat Upper Recovery Heart 2" : "Bow and can_play(Song_of_Time) and Longshot",
+            "Shadow Temple MQ After Boat Lower Recovery Heart" : "
```

Comment:
What do the recovery hearts in this room share flags with? Honestly I can't remember if I've ever seen them before.

Reply:
Good catch. 
Upper recovery heart 1 shares a flag w/ Whispering Walls Pot 2. 
Upper recovery heart 2 shares a flag w/ Truth Spinner Small Wooden Crate 4.

I think MQ Shadow has quite a few other shared flags that were overlooked. Will have to address those.